tests/app/templating/test_schema_context.py

```diff
@@ -1,11 +1,12 @@
 from unittest.mock import MagicMock
 
 from app.data_model.answer_store import AnswerStore
+from app.questionnaire.location import Location
 from app.templating.schema_context import build_schema_context
-from tests.app.framework.survey_runner_test_case import SurveyRunnerTestCase
+from tests.app.app_context_test_case import AppContextTestCase
 
 
-class TestSchemaContext(SurveyRunnerTestCase):  # pylint: disable=too-many-public-methods
+class TestSchemaContext(AppContextTestCase):  # pylint: disable=too-many-public-methods
```

Comment:
I think we need another test for the new evaluation of whether an answer is on the routing path, something like:
- Add 2 answers to the answer store with different locations
- Create a routing path that includes one answer location but not the other
- Verify the first answer in the schema context, and the second isn't